src/chunkserver/datastore/chunkserver_datastore.h

```diff
@@ -133,7 +138,21 @@ class CSMetaCache {
         chunkMap_.clear();
     }
 
+    void SetCondPtr(std::shared_ptr<std::condition_variable> cond) {
+        cvar_ = cond;
+    }
+
+    void SetSyncChunkLimits(const uint64_t limits, const uint64_t threshold) {
+        syncChunkLimits_ = limits;
+        syncThreshold_ = threshold;
+    }
+
  private:
+    uint64_t syncChunkLimits_;
+    uint64_t syncThreshold_;
```

Comment:
Why are these two variables `syncChunkLimits_`  and `syncThreshold_` contained in three classes `CSMetaCache`，`CSChunkFile` and `CopysetNode`?